Public/Src/Cache/MemoizationStore/Library/Sessions/LocalCache.cs

```diff
@@ -109,15 +124,34 @@ public class LocalCache : OneLevelCache
             IClock clock,
             ConfigurationModel configurationModel,
             SQLiteMemoizationStoreConfiguration memoConfig,
-            bool checkLocalFiles,
-            bool emptyFileHashShortcutEnabled)
+            ContentStoreSettings contentStoreSettings,
+            bool createServer,
+            int grpcPort = 0,
+            int maxQuotaMB = 0,
+            string cacheName = null,
+            string stampId = null,
+            string ringId = null,
+            string scenarioName = null)

```

Comment:
Should we add a LocalCacheSettings class or struct to handle all these default values and `base(` and `this(` calls and make it a bit more maintainable for new values going forwrad

Reply:
+100 500!!!!!!

---
In reply to: [277486273](https://github.com/Microsoft/BuildXL/pull/185#discussion_r277486273) [](ancestors = 277486273)